neuro_san/internals/network_providers/agent_network_storage.py

```diff
@@ -56,8 +57,9 @@ def add_agent_network(self, agent_name: str, agent_network: AgentNetwork):
         """
         is_new: bool = False
         with self.lock:
-            is_new = self.agents_table.get(agent_name, None) is None
+            is_new = self.agents_table.get(agent_name) is None
```

Comment:
None is not necessary, it's the default for get()
Below: Add a little whitespace for readability.